src/Razor/src/Microsoft.VisualStudio.RazorExtension/AssemblyInfo.cs

```diff
@@ -16,7 +16,38 @@
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.Extensions.Configuration.Abstractions.dll")]
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.Extensions.Logging.dll")]
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.Extensions.Logging.Abstractions.dll")]
+[assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.Extensions.Primitives.dll")]
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.Extensions.FileSystemGlobbing.dll")]
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.Extensions.FileProviders.Abstractions.dll")]
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.Extensions.FileProviders.Physical.dll")]
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\System.Reactive.dll")]
+[assembly: ProvideBindingRedirection(
```

Comment:
Should talk with WTE folks to see if this is the recommended way to have codebases/binding redirections (having both vs. having just one)